### PR TITLE
[stable/elasticsearch-exporter] Add options to set relabel configuration for metrics

### PR DESCRIPTION
**Pull request by ricoberger**
#### What this PR does / why we need it:

Add a new option to the ServiceMonitor to set a relabeling configuration. This can be used to replace the values of the `instance` label with the name of the pod.

Example:

```yaml
serviceMonitor:
  enabled: true
  relabelings:
    - action: replace
      sourceLabels:
        - __meta_kubernetes_pod_name
      targetLabel: instance
```

#### Checklist

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/elasticsearch-exporter/templates/servicemonitor.yaml [hunk 1]
```diff
@@ -25,6 +25,10 @@ spec:
     port: {{ .Values.service.metricsPort.name }}
     path: {{ .Values.web.path }}
     scheme: {{ .Values.serviceMonitor.scheme }}
+    {{- if .Values.serviceMonitor.relabelings }}
+    relabelings:
+      {{- toYaml .Values.serviceMonitor.relabelings | nindent 6 }}
```
Comment (svenmueller) [inline stable/elasticsearch-exporter/templates/servicemonitor.yaml]: would be good to have a consistent indentation (example https://github.com/helm/charts/blob/3b2db4aaf077516b74d2f3e379f4bc6a33b80d5b/stable/elasticsearch-exporter/templates/servicemonitor.yaml#L16)

Comment (ricoberger) [reply stable/elasticsearch-exporter/templates/servicemonitor.yaml]: Done.

**Comment by ricoberger** [comment]
Hi, any chance to get this merged?

**Comment by caarlos0** [comment]
actually, needs to bump the chart version...

**Comment by ricoberger** [comment]
> actually, needs to bump the chart version...

Done.

File: stable/elasticsearch-exporter/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Elasticsearch stats exporter for Prometheus
 name: elasticsearch-exporter
-version: 3.2.0
+version: 3.2.1
```
Comment (caarlos0) [inline stable/elasticsearch-exporter/Chart.yaml]: since its a new feature, should probably increase the minor instead:

```suggestion
version: 3.3.0
```

Comment (ricoberger) [reply stable/elasticsearch-exporter/Chart.yaml]: Sure.

**Pull request merged**